### ISSUE TITLE
Remove Ruby 1.9.3 from supported Rubies

### ISSUE THREAD

**Issue by dsandstrom** [sgruhier/foundation_rails_helper#111]
Tests fail: https://travis-ci.org/sgruhier/foundation_rails_helper/jobs/93031454

```
Gem::InstallError: mime-types-data requires Ruby version >= 2.0.
An error occurred while installing mime-types-data (3.2015.1120), and Bundler
cannot continue.
Make sure that `gem install mime-types-data -v '3.2015.1120'` succeeds before
bundling.
```

If there is no easy fix, support should be dropped.

**Comment by dgmstuart** [comment]
Having a look at this.

This seems relevant: http://stackoverflow.com/a/33856850/1035431

**Comment by dsandstrom** [comment]
Looks like it was fixed at Travis CI: https://github.com/travis-ci/travis-ci/issues/5145

**Comment by dgmstuart** [comment]
Cool - Looks like the branch is rebuilding. Did you kick that off somehow, or is it just coincidence??

**Comment by dsandstrom** [comment]
I restarted it.

**Comment by dsandstrom** [comment]
Didn't work. We might have to do the .travis.yml fix.

**Comment by dgmstuart** [comment]
OK - I'll give it a go

**Comment by dgmstuart** [comment]
Well that didn't work either :(

**Comment by dgmstuart** [comment]
I wonder if the containerised version will address this? trying that

**Comment by dgmstuart** [comment]
Nope :(

**Comment by dsandstrom** [comment]
Travis CI problems are annoying because you can't test locally.

**Comment by dgmstuart** [comment]
@dsandstrom Am I definitely including those lines in the .yml correctly?

So the fix is in the dpl - as far as I can understand, that's something that travis uses internally to do the builds. What I can't work out is what version travis.org is running. Could it just be that .org hasn't caught up yet?

**Comment by dgmstuart** [comment]
See https://github.com/sgruhier/foundation_rails_helper/pull/116#issuecomment-160206451

**Comment by dsandstrom** [comment]
Fixed by #117.  Thanks @dgmstuart & @JoeWoodward
